include/pocl.h

```diff
@@ -523,14 +522,17 @@ struct _cl_command_node
   /* The index of the targeted device in the **program** device list. */
   unsigned program_device_i;
   cl_int ready;
-
-  /* Fields needed by buffered commands only: */
+  /* true if the node belongs to a cl_command_buffer_khr */
+  cl_int buffered;
 
   /* Which of the command queues in the command buffer's queue list
    * this command was recorded for. */
   cl_uint queue_idx;
   /* List of buffers this command accesses, used for inserting migrations */
   pocl_buffer_migration_info *migr_infos;
+
+  /* pointer to command buffer or NULL */
```

Comment:
Pls instead describe when non-NULL (when added to a command buffer - implying there can be only one per our command struct).